src/Files.App/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -155,13 +155,13 @@
 					Width="Auto"
 					MinWidth="40"
 					AccessKey="H"
-					Command="{x:Bind ViewModel.Share, Mode=OneWay}"
-					IsEnabled="{x:Bind converters:MultiBooleanConverter.AndConvert(ViewModel.CanShare, ViewModel.InstanceViewModel.CanShareInPage), Mode=OneWay, FallbackValue=False}"
-					Label="{helpers:ResourceString Name=BaseLayoutItemContextFlyoutShare/Text}"
+					Command="{x:Bind Commands.ShareItem, Mode=OneWay}"
+					IsEnabled="{x:Bind Commands.ShareItem.IsExecutable, Mode=OneWay, FallbackValue=False}"
+					Label="{x:Bind Commands.ShareItem.Label}"
 					LabelPosition="Collapsed"
-					ToolTipService.ToolTip="{helpers:ResourceString Name=BaseLayoutItemContextFlyoutShare/Text}">
+					ToolTipService.ToolTip="{x:Bind Commands.ShareItem.Label}">
```

Comment:
It might be better to use the description in the tooltip

Reply:
Label is correct because it'll show the shortcut as well.